web/components/login/Login.js

```diff
@@ -9,11 +9,15 @@ class Login extends React.Component {
   constructor(props) {
     super(props);
 
+    // Deconstruct props and state
+    const { isLoaded, isEmpty, firebase, auth } = this.props;
+    const { email, password } = this.state;
+
     this.state = {
       email: '',
       password: '',
-      loading: !this.props.auth.isLoaded,
-      loggedIn: !this.props.auth.isEmpty,
+      loading: !isLoaded,
```

Comment:
We specifically want to check if `auth` is loaded; not sure if checking if the `props` is `isLoaded` encompasses whether `auth` is loaded. 

Regardless I think it would be best to be explicit here? by calling `isLoaded(auth)` and import `isLoaded` from `react-redux-firebase`

Reply:
I want to avoid making any functional changes in this PR - here, `!isLoaded` does the same thing as what was there before (`!this.props.auth.isLoaded`)

Reply:
Sorry, i meant that `isLoaded` is not the same as what it was before. In the change above, you deconstructed `isLoaded` from `this.props` - meaning `isLoaded = this.props.isLoaded`. But we want it to be `isLoaded = this.props.auth.isLoaded`. 

So we should define `isLoaded` as `const { isLoaded } = auth` (or using some other ES6 syntax - i'm not too familiar with them) after line 11

Reply:
OH

Reply:
mb see 21b0c48